packages/cli/configuration/src/docs-yml/getAllPages.ts

```diff
@@ -17,7 +17,7 @@ async function loadBatch(
     const pairs = await Promise.all(
         files.map(async (file) => {
             const content = await readFile(file, "utf-8");
-            return [relativize(absolutePathToFernFolder, file), content];
+            return [await relativize(absolutePathToFernFolder, file), content];
```

Comment:
should eslint have caught this?

Reply:
not 100% sure. `Object.fromEntries(pairs);` might have removed eslint's ability to detect a typebreak